### ISSUE TITLE
[css-align] Circular center alignment

### ISSUE THREAD

**Issue by Crissov** [w3c/csswg-drafts#1849]
* https://drafts.csswg.org/css-align/#positional-values
* https://medium.muz.li/optical-effects-9fca82b4cd9a#8a18

For `center` alignment, matching the center of the smallest possible virtual circle (or possibly ellipse) surrounding the box to align with the same center of its alignment container may give better visual results than using the horizontal and vertical center of rectangular boxes. I propose a new positional alignment keyword `circular-center` for this.

**Comment by Crissov** [comment]
Alternatively, a new property could be introduced to specify the “gravitational” center of a box which is used for (`center`) alignment.

**Comment by tabatkins** [comment]
I'm not sure I understand what's being requested here. For rectangles, the circumcircle has the same center as the rectangle itself.

The medium post you've linked to describes the use of a circumcircle around the *non-rectangular shape*, which can definitely give a different answer than the center of the shape's bounding box. But we only have rectangles to work with.

**Comment by Crissov** [comment]
Yes, that is why I referenced the Shapes module, although its `shape-outside` currently only affects the [_float area_](https://drafts.csswg.org/css-shapes/#float-area) of float elements (defaulting to the [margin box](https://drafts.csswg.org/css-box/#margin-box)) and not the [_alignment subject_](https://drafts.csswg.org/css-align/#alignment-subject) (also defaulting to the margin box).

**Comment by Crissov** [comment]
Designers may also want this effect for boxes that have non-rectangular [`border-shape`](https://drafts.csswg.org/css-borders-4/#border-shape) applied. 

PS: Maybe this could also be solved by a keyword like `shape-center` (similar to [`anchor-center`]).

 [`anchor()`]: <https://drafts.csswg.org/css-anchor-position/#anchor-pos>
 [`anchor-size()`]: <https://drafts.csswg.org/css-anchor-position/#anchor-size-fn>
 [`anchor-center`]: <https://drafts.csswg.org/css-anchor-position/#anchor-center>
